### PR TITLE
Add PrimaryButton component

### PR DESCRIPTION
**Pull request by im36-123**
Add PrimaryButton component.

I will create Secondary and Danger button components after merging this pull request.

### CODE REVIEW

File: src/components/Button/PrimaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+      box-sizing: border-box;
+      cursor: pointer;
+      transition: ${isTouchDevice ? 'none' : `all ${interaction.hover.animation}`};
+
+      &.s {
+        min-width: 80px;
+        height: 32px;
+        padding: 0 ${size.pxToRem(size.space.XXS)};
+        line-height: 32px;
+      }
```
Comment (ouji-miyahara) [inline src/components/Button/PrimaryButton.tsx]: font-size of `small button` is 11px in design

Comment (ouji-miyahara) [reply src/components/Button/PrimaryButton.tsx]: and `small button ` has 27px height in design 🙇

File: src/components/Button/PrimaryButton.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+import { hoverable } from '../../hocs/hoverable'
+import { isTouchDevice } from '../../libs/ua'
+
+type Tag = 'button' | 'a'
+type Size = 's'
```
Comment (nabeliwo) [inline src/components/Button/PrimaryButton.tsx]: I feel uncomfortable that the size is only 's'.

If the value of Size is only 's', props should be 'small' to accept a boolean.

If you keep props as Size, you should accept 'default' or 'small' and set 'default' as defaultValue.
And in style, you should separate the implementation with `.default` and `.small`

File: src/components/Button/PrimaryButton.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      min-width: 2rem;
+      height: 40px;
+      line-height: 40px;
+      vertical-align: middle;
+      border: none;
+      border-radius: ${frame.border.radius.m};
+      background-color: ${palette.MAIN};
+      color: #fff;
+      font-size: ${size.pxToRem(size.font.TALL)};
+      font-weight: bold;
```
Comment (nabeliwo) [inline src/components/Button/PrimaryButton.tsx]: Do you need `font-weight: bold`?

Comment (im36-123) [reply src/components/Button/PrimaryButton.tsx]: Yes, design was defined as font-weight: w6.

**Comment by im36-123** [comment]
Thank you for your feedback.
Please check again 🙏

**Comment by im36-123** [comment]
Thank you for your great reviews !!
I will merge this branch, after resolve conflicts.

**Pull request merged**